### PR TITLE
Chrome 22 added `webextensions.manifest.commands` support

### PR DESCRIPTION
**Pull request by queengooborg**
This PR updates and corrects version values for Chromium (Chrome, Opera, Samsung Internet, WebView Android) for the `commands` Web Extensions manifest property. The data comes from a commit in the browser's source code, mapped to a version number using available tooling or via the commit timestamp.

Commit: https://chromiumdash.appspot.com/commit/8e1b9cf4b212c6596d979b333c1627d6acb714e2

### CODE REVIEW

**Review by rebloor — COMMENTED** [review]
@queengooborg `_execute_action` should remain at 88 - the MV3 features landed sooner but the release was 88. Also, `_execute_browser_action` and `_execute_page_action` will have been earlier - most likely with the action and page action APIs (but I have no more information).

**Comment by queengooborg** [comment]
What should we do for `_execute_browser_action` and `_execute_page_action` then?  We could set them as ranges, though I'm working to eliminate to ranges (and focusing on webextensions to start)!

**Comment by rebloor** [comment]
@dotproto Any thoughts on how we find when `_execute_browser_action` and `_execute_page_action` were implemented in Chrome?

**Comment by danielhjacobs** [comment]
At least as early as September 18, 2012, https://chromium.googlesource.com/chromium/src/+/09d2e1d94dab63e571d527fb716c0fc86dd51206/chrome/common/extensions/docs/templates/intros/commands.html referenced _execute_browser_action and _execute_page_action (when I hover over "12 years ago" on https://chromium.googlesource.com/chromium/src/+log/09d2e1d94dab63e571d527fb716c0fc86dd51206/chrome/common/extensions/docs/templates/intros/commands.html, it says that date).

That was before the Chrome release which added support for pageAction, version 23: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/pageAction#browser_compatibility

**Comment by danielhjacobs** [comment]
In https://chromium.googlesource.com/chromium/src/+log/854e828ba4af79eec2d97678264b97fa38a41ecd/chrome/common/extensions/docs/server2/templates/intros/commands.html, on August 30, 2012, this file inside the folder server2  already listed _execute_browser_action and _execute_page_action.

Even before then in https://chromium.googlesource.com/chromium/src/+/4a7ce7a4c35f99571d6989b00c492149dbc2ff31/chrome/common/extensions/docs/server2/templates/intros/experimental_commands.html from July 31, 2012, the file there already listed this, though at the time it was named experimental_commands.

Even before then in https://chromium.googlesource.com/chromium/src/+/3c0d22621d6f2fbcfc4d1c02172ff73fe5c6f89e/chrome/common/extensions/docs/server2/templates/intros/experimental_keybinding.html from July 10, 2012, the file there already listed this, though at the time it was named experimental_keybinding.

Even before then in https://chromium.googlesource.com/chromium/src/+/d4468be582737a9c9d8f9112735604c2610f9afd/chrome/common/extensions/docs/server2/templates/private/experimental_keybinding_intro.html from July 6, 2012, the file there already listed this, though at the time it was named experimental_keybinding_intro.html.

**Comment by danielhjacobs** [comment]
Since commands itself wasn't listed as supported until Chrome 54, I guess my assumption would be that the moment it became supported it already supported these commands.

**Comment by danielhjacobs** [comment]
https://issues.chromium.org/issues/40197373. A bug referencing _execute_browser_action at least as early as April 2012.

**Comment by danielhjacobs** [comment]
I finally found the relevant info in the docs:

https://web.archive.org/web/20131228200953/https://developer.chrome.com/extensions/commands.html

"Stable since Chrome 25"

Already listed _execute_browser_action and _execute_page_action at the time.

File: webextensions/manifest/commands.json [hunk 1]
```diff
@@ -220,7 +220,7 @@
           "__compat": {
             "support": {
               "chrome": {
-                "version_added": true,
+                "version_added": "86",
```
Comment (danielhjacobs) [inline webextensions/manifest/commands.json:223]: This should probably be 25, along with commands itself (which is currently ≤54). See https://web.archive.org/web/20130824074541/https://developer.chrome.com/extensions/commands.html

File: webextensions/manifest/commands.json [hunk 2]
```diff
@@ -247,7 +247,7 @@
...
```
Comment (danielhjacobs) [inline webextensions/manifest/commands.json:250]: This should probably be 25, along with commands itself (which is currently ≤54). See https://web.archive.org/web/20130824074541/https://developer.chrome.com/extensions/commands.html

**Review by danielhjacobs — CHANGES_REQUESTED** [review]

**Comment by caugner** [comment]
@danielhjacobs Nice research, thanks! Fixed in [0fa5fbb](https://github.com/mdn/browser-compat-data/pull/24293/commits/0fa5fbbc4b9bd5c1144c2d4ca2fe7242ee3bf2ef).

**Comment by caugner** [comment]
@rebloor @queengooborg Any concerns with the evidence found by @danielhjacobs?

**Review by caugner — APPROVED** [review]

**Review by rebloor — APPROVED** [review]
No concerns on my part

**Review by danielhjacobs — APPROVED** [review]

**Pull request merged**